detect_secrets/filters/heuristic.py

```diff
@@ -191,3 +191,15 @@ def is_lock_file(filename: str) -> bool:
         'Podfile.lock',
         'yarn.lock',
     }
+
+
+def is_swagger_file(filename: str) -> bool:
+    """
+    Filters swagger files and paths, like swagger-ui.html or /swagger/.
+    """
+    return bool(_get_swagger_regex().search(filename))
+
+
+@lru_cache(maxsize=1)
+def _get_swagger_regex() -> Pattern:
+    return re.compile(r'.*swagger.*')
```

Comment:
I think this is probably too aggressive? Looks like it introduces a huge slowdown again with the `rfc.model`.

Reply:
No, I think there is no bug here. The branch `filter/is-swagger-file` has been created from the detect-secrets master before you merge #420. Now #420 is merged, and the bug detected in the `rfc.model` scanning is solved. I have updated the Pull Request with the latest changes in master, so it should work fine now.
Please, check it again. Thank you very much!

Reply:
ohh, gotcha. You're probably right.